docs/_includes/migrate-from-msword.adoc

```diff
@@ -0,0 +1,234 @@
+* Fix tables - merged cells will have unmerged, column widths need putting back.
+. Try to convert it, and fix any errors that come up.
+// pandoc supposedly only uses UTF-8, and the xml file is windows encoded, but I haven't found any problems so far.
+// You definitely do get encoding errors if you go via HTML.
++
+NOTE: Using this route, there _shouldn't_ be any encoding errors.
+If there are, use the Ant `nativetoascii` task to make them visible as \NNNN codes.
+
+// Examples are in Ant because that is what I use, an I don't know what else Windows has that can do this.
+// Should I try to replicate it in something else?
+// Should I give a cut-and-paste script? - No, it would probably cause lots of "it dont work" complaints. 
+// Doing it in fragments makes it clear that it needs customising.
+
+These {url-ant}[Apache Ant] task fragments automate some of these steps.
+
+Extract image files:
+
+TIP: docx is just a zip file with a docx file extension.
+The images are always in the same place.
+
+// Gets images from canvases as well, but not vector graphics
+[source,xml]
+----
+<unzip src="${infile}" dest="build"/>
+<copy todir="dist/images">
+  <fileset dir="build/word/media"/>
+</copy>
+----
+
+Run Pandoc:
+
+[source,xml]
+----
+<exec executable="pandoc">
+    <arg value="-f"/><arg value="docx"/>
+    <arg value="-t"/><arg value="asciidoc"/>  
+    <arg value="--wrap=none"/>
+    <arg value="--atx-headers"/>
+    <arg value="--normalize"/>
+    <arg value="-o"/><arg value="${outfile}"/>
+    <arg value="${infile}"/>
+</exec>
+----
+
+Automate some of the editing steps (adjust the regexps to match your particular document):
+
+[source,xml]
+----
+<!-- Delete automatically inserted ids -->
+<replaceregexp
+    file="${outfile}" 
+    match="\[\[_.*]]"
+    replace=""
+    flags="g"/>
+
+<!-- Shorten table delimiters -->
+<replaceregexp
+    file="${outfile}"
+    match="\|==*"
+    replace="|===="
+    flags="g"/>
+
+<!-- 1 sentence per line. Be careful not to match lists. It will get confused by abbreviations, but there is no way around that. -->
+<replaceregexp
+    file="${outfile}"
+    match="(\w\w+)\.\s+(\w)"
+    replace="\1.${line.separator}\2"
+    flags="g"
+    byline="true"/>
+
+<!-- Replace figure captions with id and title -->
+<replaceregexp
+    file="@{outfile}"
+    match="^Figure (\d?)*\s?(.*)"
+    replace="[[fig-\1]]${line.separator}.\2${line.separator}"
+    byline="true"/>
+
+<!-- Replace references to figures with asciidoc xref -->
+<replaceregexp
+    file="@{outfile}"
+    match="Figure (\d?)"
+    replace="&lt;&lt;fig-\1&gt;&gt;"
+    flags="g"/>
+----
+
+=== Google Docs
+Google Docs can already upload and edit MS Word docx files.
+With this addon from Guillaume Grossetie: {url-google-asciidoc}[AsciiDoc Processor]
+you can copy and paste part or all of the document from Google Docs as AsciiDoc text. 
+The features that it can handle seem to be slightly fewer than Pandoc, but expect further development.

```

Comment:
We should probably say "substantially fewer" and that this tool is still in the early stages. We don't want to provide false hope and have people disappointed. In time, we can strengthen the message.